arm/Microsoft.Network/virtualNetworks/readme.md

```diff
@@ -10,6 +10,7 @@ This template deploys a Virtual Network (vNet) with 2 optional Subnets.
 | `Microsoft.Authorization/roleAssignments` | 2020-04-01-preview |
```

Comment:
Regenerate with function `utilities\tools\Set-ModuleReadMe.ps1`. You may need to merge the latest main back into your branch to get the latest, fixed version

Reply:
Already did it. Thanks to Marius